packages/webpack/src/config/client.js

```diff
@@ -182,9 +182,9 @@ export default class WebpackClientConfig extends WebpackBaseConfig {
       `${querystring.stringify(hotMiddlewareClientOptions)}&path=${clientPath}`.replace(/\/\//g, '/')
 
     // Entry points
-    config.entry = {
+    config.entry = Object.assign({}, config.entry, {
```

Comment:
Would you please use rest operator here? `{ ...config.entry, `

Reply:
I wanted to be consistent with existing style:

https://github.com/nuxt/nuxt.js/blob/89f88669c10ce9100b109749aeab445638b851a8/packages/webpack/src/config/server.js#L74-L77

Reply:
That's ok. will refactor later :)